extensions/control-plane/store/sql/contract-negotiation-store-sql/src/test/java/org/eclipse/edc/connector/store/sql/contractnegotiation/SqlContractNegotiationStoreExtensionTest.java

```diff
@@ -40,18 +45,25 @@ class SqlContractNegotiationStoreExtensionTest {
 
     @Test
     void initialize(ServiceExtensionContext context, ObjectFactory factory) {
+        var ctx = spy(context);
```

Comment:
not needed, the context it's already wrapped by `spy` by `DependencyInjectionExtension`